src/components/DeckPreview.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+import AgeText from './AgeText';
+
+class DeckPreview extends React.Component {
+  constructor(props) 
+  {
+      super(props);
+  }
+
+  render() {
+    var deck = this.props.deck;
```

Comment:
I've been trying to stick to `const`/`let` scoping in all the React components. `var` uses weird, archaic scoping rules.
```suggestion
    const { deck } = this.props;
```